### PR TITLE
Initial draft compliance reqs

### PR DESCRIPTION
**Pull request by cstrangedk**

### CODE REVIEW

File: validators/compliance-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
...le_admin", "state": {"Status": "running", "Running": true, "Paused": false, "Restarting": false, "OOMKilled": false, "Dead": false, "Pid": 21192, "ExitCode": 0, "Error": "", "StartedAt": "2021-04-23T16:36:27.127219597Z", "FinishedAt": "0001-01-01T00:00:00Z", "Health": {"Status": "healthy", "FailingStreak": 0, "Log": [{"Start": "2021-06-08T13:25:52.647447529Z", "End": "2021-06-08T13:25:52.753230615Z", "ExitCode": 0, "Output": "Modification time diff: 22.64755392074585, limit is 600\n"}, {"Start": "2021-06-08T13:26:52.758628839Z", "End": "2021-06-08T13:26:52.88566706Z", "ExitCode": 0, "Output": "Modification time diff: 6.810468435287476, limit is 600\n"}, {"Start": "2021-06-08T13:27:52.891066058Z", "End": "2021-06-08T13:27:52.993667657Z", "ExitCode": 0, "Output": "Modification time diff: 5.668185472488403, limit is 600\n"}, {"Start": "2021-06-08T13:28:52.998967902Z", "End": "2021-06-08T13:28:53.102507618Z", "ExitCode": 0, "Output": "Modification time diff: 1.5429127216339111, limit is 600\n"}, {"Start": "2021-06-08T13:29:53.108332234Z", "End": "2021-06-08T13:29:53.204775561Z", "ExitCode": 0, "Output": "Modification time diff: 5.435116291046143, limit is 600\n"}]}}}], "error": null}%`
```
Comment (ebruengwall-ant) [inline validators/compliance-requirements.adoc]: I feel like nobody would pay attention to this json data.

File: validators/compliance-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,110 @@
+= Validator Node Compliance Requirements
+
+The following document outlines compliance requirements for SKALE Network nodes. Failure to follow these requirements will result in a node forfeiting bounty for each month that it's not in compliance. Please be sure to follow the entire proper setup and backup procedures described in Validator Denali Upgrade documentation to ensure your node is operating as intended.
+
+== Compliance Requirements
+
+The following compliance requirements are required to ensure all nodes meet a standard of performance. 
+
+:warning: Failure to meet ANY ONE of the following Denali compliance requirements will result in a node being marked “Out of Compliance” by the NODE Foundation and the node will be forced to forfeit monthly bounties until it has met compliance requirements.
```
Comment (ebruengwall-ant) [inline validators/compliance-requirements.adoc]: How much time do they have to fix their issues? until it has met compliance doesn't give enough information. In other words there has to be a limit for downtime or not meeting the requirements

Comment (ebruoguzberk) [reply validators/compliance-requirements.adoc:9]: And when do we set this flag?

File: validators/compliance-requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+:note: Compliance requirements may be reviewed and modified at any given time.
```
Comment (ebruengwall-ant) [inline validators/compliance-requirements.adoc:11]: but it doesn't mean they forfeit immediately. If we set this flag 1 day earlier than the get bounty they forfeit but if it happens the first week of the month they have 3 weeks to fix the issue.

File: validators/compliance-requirements.adoc [hunk 4]
```diff
@@ -0,0 +1,110 @@
+= Validator Node Compliance Requirements
```
Comment (ebruengwall-ant) [inline validators/compliance-requirements.adoc:1]: General idea: We also should add telegram notification to not in compliance/compliance so they will get notified. Or give an example here where can they get this data from solidity so they will add it to their bots.

**Review by ebruengwall-ant — DISMISSED** [review]

**Review by ebruengwall-ant — APPROVED** [review]

**Pull request merged**